lib/crawler.js

```diff
@@ -381,12 +385,15 @@ Crawler.prototype.getRequestOptions = function (queueItem) {
         agent: agent,
         headers: {
             "Accept": "text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8",
-            "Accept-Encoding": "gzip, deflate",
```

Comment:
Yes, I added that in d5d53a1a2559f90cac96d8abb384b78fe64fb2fa because I thought we actually did decompress responses based on their headers, but turns out that was only done for the discoverResources method.

That change hasn't made it into a release yet, but I think that setting `decompressResponses=true` as a default is smart.